### PR TITLE
Feature/willji0023/interview routes

### LINKED ISSUE #4: Interview schema and routes

### PR DESCRIPTION
**Pull request by willji0023**
### Administrative Info
[GitHub Issue #4]

### Changes
- Implemented create and get routes for interviews

### Testing
- Tested routes using Insomnia and made sure objects were being created on a local MongoDB

### Confirmation of Change 
Run `npm start`
Initiate HTTP requests to:
- `GET http://localhost:8000/api/interviews/:interviewId?stage=:stage`
- `POST http://localhost:8000/api/interviews`

### CODE REVIEW

File: models/interview.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      type: String,
+      required: true,
+    },
+    location: {
+      type: String,
+      required: true,
+    },
+    internalNotes: {
+      type: String,
+      required: true,
```
Comment (petabite) [inline models/interview.js]: @dnanjunda Do we want `internalNotes` to be an optional field here as well?

**Review by petabite — APPROVED** [review]
Nice work! We might want to verify that the `user` and `ambassador` ids exist before we create the `Interview`, but we can do that once those models have been implemented (might be a good idea to add a TODO comment for the future if you'd like).

File: models/interview.js (hunk 1 above)
Comment (willji0023) [reply models/interview.js]: Sounds good :+1: fixed!

**Review by dnanjunda — APPROVED** [review]
LGTM. Nice job William!

**Pull request merged**